File: ueueo-framework/ueueo-localization/src/main/java/Abp/Localization/LocalizationResourceContributorList.cs
using System.Collections.Generic;
using System.Linq;
using Microsoft.Extensions.Localization;

namespace Volo.Abp.Localization;

public class LocalizationResourceContributorList : List<ILocalizationResourceContributor>
{
    public LocalizedString GetOrNull(String cultureName, String name)
    {
        for (var contributor in this.Select(x => x).Reverse())
        {
            var localString = contributor.GetOrNull(cultureName, name);
            if (localString != null)
            {
                return localString;
            }
        }

        return null;
    }

    public void Fill(String cultureName, Dictionary<String, LocalizedString> dictionary)
    {
        for (var contributor in this)
        {
            contributor.Fill(cultureName, dictionary);
        }
    }
}

File: ueueo-framework/ueueo-localization/src/main/java/com/ueueo/localization/LocalizationResourceContributorList.java
package com.ueueo.localization;

import java.util.ArrayList;
import java.util.Map;

public class LocalizationResourceContributorList extends ArrayList<ILocalizationResourceContributor> {

    public LocalizedString getOrNull(String cultureName, String name) {
        for (ILocalizationResourceContributor contributor : this) {
            LocalizedString localString = contributor.getOrNull(cultureName, name);
            if (localString != null) {
                return localString;
            }
        }

        return null;
    }

    public void fill(String cultureName, Map<String, LocalizedString> dictionary) {
        for (ILocalizationResourceContributor contributor : this) {
            contributor.fill(cultureName, dictionary);
        }
    }
}
